examples/src/examples/AfiInteractiveOptimizerEngineExample.java

```diff
@@ -108,12 +112,28 @@ public static void main(String[] args) throws InterruptedException, JsonProcessi
       }
       
       ObjectMapper mapper = new ObjectMapper();
-      String json = mapper.writeValueAsString(tables);
-      System.out.println(json); // Prints the result in 2D table format.
+      // Prints the results in 2D table format.
+      for (TableData table : tables) {
+        List<Row> rows = table.getRows();
+        String json = mapper.writeValueAsString(rows);
+        System.out.println(json);
+      }
+      // Prints the metadata
+      for (TableData table : tables) {
+        if (table.getMetadata().size() > 0) System.out.println("Printing metadata...");
```

Comment:
Is it possible to use table.getRawMetadata().size() ?
The next for loop is on rawMetadata so it would make sense

Reply:
yup, good point! fixed https://github.com/factset/analyticsapi-engines-java-sdk/pull/65/commits/817903b5016a8879c78d770c2e6a51dd4cfd1235